hypha/core/templates/components/scroll-to-top.html

```diff
@@ -0,0 +1,14 @@
+{% load heroicons %}
+
+<template x-teleport="body" x-data="{scrollBackTop: false, lastScrollTop: 0}">
+    <button
+        x-show="scrollBackTop"
+        x-transition
+        x-transition.duration.500ms
+        x-on:scroll.window.throttle.50ms="scrollBackTop = (window.pageYOffset < lastScrollTop && window.pageYOffset > window.outerHeight * 0.4) ? true : false; lastScrollTop = window.pageYOffset;"
+        @click="window.scrollTo({top: 0, behavior: 'smooth'})"
+        aria-label="Back to top"
+        class="fixed top-0 right-1/2 px-3 py-2 mt-10 -mr-[64px] text-white bg-light-blue/80 hover:bg-light-blue hover:text-white hover:shadow-lg transition-all focus:outline-none cursor-pointer shadow-lg rounded-2xl">
```

Comment:
```suggestion
        class="fixed top-0 right-1/2 px-3 py-2 mt-10 -mr-[64px] text-white bg-light-blue/80 z-30 hover:bg-light-blue hover:text-white hover:shadow-lg transition-all focus:outline-none cursor-pointer shadow-lg rounded-2xl">
```
Did user testing for this today. The only issue I found was it would sometimes get tucked behind/cut off by labels on the submission view. A z-index increase seemed to fix that when I was playing with it. Other than that it's ready for live!

![Screenshot 2023-12-01 at 10 20 57 AM](https://github.com/HyphaApp/hypha/assets/145372368/15bf2c50-0003-48e1-b4c1-e3dd0e56a68d)
![Screenshot 2023-12-01 at 10 07 26 AM](https://github.com/HyphaApp/hypha/assets/145372368/1a224908-1fcb-4ef1-b2c0-e203f973754d)

Reply:
Thank you!